requirements.txt

```diff
@@ -1,4 +1,4 @@
-elastic-apm==6.19.0
+elastic-apm==6.23.0
```

Comment:
@zmoog I had to bump this dependency to get everything working end to end. I think this is due to python 3.12 bump

Reply:
What kind of error did you get?